hack/runtime-migrator/cmd/crb-cleanup/config.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"log/slog"
+	"os"
+	"reflect"
+	"strings"
+)
+
+type Config struct {
+	Kubeconfig string
+	DryRun     bool
+	Verbose    bool
+	Force      bool
+	OldLabel   string
+	NewLabel   string
+	Output     string
+}
+
+func ParseConfig() Config {
+	cfg := Config{}
+	flag.StringVar(&cfg.Kubeconfig, "kubeconfig", "", "Kubeconfig file path")
+	flag.StringVar(&cfg.OldLabel, "oldLabel", LabelSelectorOld, "Label marking old CRBs")
+	flag.StringVar(&cfg.NewLabel, "newLabel", LabelSelectorNew, "Label marking new CRBs")
+	flag.StringVar(&cfg.Output, "output", "", "Output folder for created files. Can also contain file prefix, if it doesn't end with `/` (can be a folder, eg ./foo/)")
+	flag.BoolVar(&cfg.DryRun, "dry-run", false, "Don't remove CRBs, write what would be removed  to ./removed.json")
+	flag.BoolVar(&cfg.Verbose, "verbose", false, "Increase the log level to debug (default: info)")
+	flag.BoolVar(&cfg.Force, "force", false, "Force remove CRBs without checking migration status")
```

Comment:
I'm not convinced to have this `force` option. IMO it's risky.

Reply:
It was specifically listed in requirements by @tobiscr, IIRC